sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/StringUtils.scala

```diff
@@ -66,12 +66,6 @@ object StringUtils extends Logging {
     "(?s)" + out.result() // (?s) enables dotall mode, causing "." to match new lines
   }
 
-  /**
```

Comment:
Is this the function that should be deleted in https://github.com/apache/spark/pull/42184?

Reply:
Sorry, I can not answer your question since I wasn't there.

Reply:
both `org.apache.spark.sql.catalyst.util.StringUtils` and `getHexString` are public, should we keep them?

Reply:
The `catalyst` package is private by contract